main.cpp

```diff
@@ -0,0 +1,104 @@
+#include <iostream>
+#include <array>
+#include <random>
+#include <chrono>
+#include <algorithm>
+#include "Card.h" // creates the card object
+#include "Deck.h" // creates the Deck object
+#include "Player.h" //creates the player object
+
+bool playerHits()
+{
+    char x;
+    std::cout << "Would you like to hit (h) or stand (s)?\n";
+    std::cin >> x;
+
+    if (x == 's') 
+        return false;
+    return true;
```

Comment:
No handling invalid input - if someone missed the 's' key and pressed 'a', it would count as a 'h'.